src/middleware.ts

```diff
@@ -90,7 +90,7 @@ const getCspHeader = (nonce: string) => {
       process.env.NODE_ENV === 'production' ? '' : `'unsafe-eval'`
     };
     style-src 'self' 'nonce-${nonce}';
-    img-src *;
+    img-src * blob: data: www.ibm.com/;
```

Comment:
Isn't the wildcard enough, or why is the www.ibm.com/ domain specified as well? :-)